mm2src/mm2_db/src/indexed_db/drivers/upgrader.rs

```diff
@@ -18,6 +18,17 @@ pub enum OnUpgradeError {
     ErrorOpeningTable { table: String, description: String },
     #[display(fmt = "Error occurred due to creating the '{}' index: {}", index, description)]
     ErrorCreatingIndex { index: String, description: String },
+    #[display(
+        fmt = "Upgrade attempt to unsupported version: {}, old: {}, new: {}",
```

Comment:
`Upgrade attempt for unsupported version: {}, old: {}, new: {}`

Reply:
`to` is the correct preposition here because we are trying to upgrade to a new version. But the sentence lacks an article here, so I will do a fix :slightly_smiling_face: